src/posts/interfaces/createPost.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface CreatePost {
```

Comment:
IMO, I think the file name and the interface name are somehow misleading?
As I understand, we're here creating the expected response after getting a request of creation of new post, but the name `CreatePost` doesn't show that this is what's expected to response with,
I mean it would be better to be called `CreatePostResponse` or anything else that would be more declarative.

Reply:
I guess it is controversial. However, there is no misleading here, as we have DTOs for body requests, and interfaces for responses.